lib/data/rates/energy_grants_calculator.yml

```diff
@@ -0,0 +1,3 @@
+- start_date: 2015-04-06
+  end_date: 3000-01-01
```

Comment:
> ... as I can't guarantee the rates will be updated by the end of the tax year I don't want to run the risk of users not being aware that they "could" make a claim for payment, so set the end date to the "end of time".

You can safely set the end date to the end of the tax year. [This line in `RatesQuery#rates`](https://github.com/alphagov/smart-answers/blob/master/lib/smart_answer/calculators/rates_query.rb#L21) ensures that the latest set of rates will be used in the case where no specific rates can be found.